test/test.go

```diff
@@ -54,6 +54,20 @@ func Drag(c fyne.Canvas, pos fyne.Position, deltaX, deltaY int) {
 	o.(fyne.Draggable).DragEnd()
 }
 
+// FocusNext focuses the next focusable on the canvas.
+func FocusNext(c fyne.Canvas) {
+	if tc, ok := c.(*testCanvas); ok {
```

Comment:
Should we log an error if it's not a testCanvas?
Otherwise people using this test code outside of the test context might be confused about the lack of effect.

Reply:
I thought of it but it’s probably YAGNI.

Reply:
I now changed it to fail :).

Reply:
I don't know if this is a nit or not - but that's not going to fail, that's going to panic!

Reply:
Ah, okay … for me that’s kind of failing :).